Setup/DownloadExternals.cmd

```diff
@@ -2,8 +2,8 @@
 
 REM Also update in Product.wxs and UI\RequiredSoftwareDlg.wxs
 
-IF NOT EXIST "%~p0\..\bin\Git-2.10.1-32-bit.exe" (
-    "%~p0\tools\curl.exe" -L -k -o ..\bin\Git-2.10.1-32-bit.exe https://github.com/git-for-windows/git/releases/download/v2.10.1.windows.1/Git-2.10.1-32-bit.exe
+IF NOT EXIST "%~p0\..\bin\Git-2.12.2-64-bit.exe" (
```

Comment:
When visiting the [windows git download page](https://git-scm.com/download/win) it automatically starts a download of the git. (I visit it once in a while to update git my self, not only trusting the git installed by git ext). On my machine the download page downloads the 64-bit version. If I run the 64-bit installer it will leave the 32-bit git that git extensions downloaded leading to confusion about what git is actually used.